cmd/notary/tuf.go

```diff
@@ -533,7 +533,7 @@ func tokenAuth(trustServerURL string, baseTransport *http.Transport, gun string,
 	}
 
 	ps := passwordStore{anonymous: readOnly}
-	tokenHandler := auth.NewTokenHandler(authTransport, ps, gun, "push", "pull")
```

Comment:
It seems like we can just make the change here to just include `"pull"` when `readonly` is true, and `"push", "pull"` when `readonly` is false?  That way nothing else needs to know about what the values of the actions have to be.